drivers/include/mtd.h

```diff
@@ -101,6 +101,27 @@ struct mtd_desc {
                 uint32_t addr,
                 uint32_t size);
 
+    /**
+     * @brief   Read from the Memory Technology Device (MTD) using
+     *          pagewise addressing.
+     *
+     * @p offset should not exceed the page size
+     *
+     * @param[in]  dev      Pointer to the selected driver
+     * @param[out] buff     Pointer to the data buffer to store read data
+     * @param[in]  page     Starting page address
```

Comment:
I find this a bit ambiguous, shouldn't this be?
```suggestion
     * @param[in]  page     Page number to start reading from
```